pom.xml

```diff
@@ -321,7 +321,7 @@
         <version.org.eclipse.jdt.core.compiler>4.6.1</version.org.eclipse.jdt.core.compiler>
         <version.org.eclipse.microprofile.config.api>1.3</version.org.eclipse.microprofile.config.api>
         <version.org.eclipse.microprofile.health.api>2.0.1</version.org.eclipse.microprofile.health.api>
-        <version.org.eclipse.microprofile.metrics.api>2.0.2</version.org.eclipse.microprofile.metrics.api>
+        <version.org.eclipse.microprofile.metrics.api>2.2.0</version.org.eclipse.microprofile.metrics.api>
```

Comment:
The jobs fail because MP Metrics version should be `2.2` instead of `2.2.0` (see https://mvnrepository.com/artifact/org.eclipse.microprofile.metrics/microprofile-metrics-api).
I've asked confirmation that this `2.2` version is correct and not a mistake (afaik it should follow the standard x.y.z scheme)

Reply:
`2.2` is the correct version as confirmed by https://github.com/eclipse/microprofile-metrics/issues/482